src/dvclive/live.py

```diff
@@ -388,8 +393,15 @@ def log_artifact(
         if self._dvc_repo is not None:
             from dvc.repo.artifacts import name_is_compatible
 
+            if copy:
+                basename = os.path.basename(os.path.normpath(path))
+                output_dir = Path(self.artifacts_dir)
+                output_dir.mkdir(exist_ok=True, parents=True)
+                shutil.copy2(path, output_dir / basename)
```

Comment:
We should also support directories here or, if we don't want to, log warning and not fail

Reply:
yep, my bad - I though it's supported by `copy2`. Looking into this ...